LoopKit/InsulinKit/DoseType.swift

```diff
@@ -15,6 +15,21 @@ public enum DoseType: String, CaseIterable {
     case resume
     case suspend
     case tempBasal
+    
+    var localizedDescription: String {
+        switch self {
+        case .basal:
+            return NSLocalizedString("Basal", comment: "localized term for basal dose type")
```

Comment:
[nit] Most of the localized string comments start with a capitalized word and don't mention anything about localization. Perhaps just "Title for basal dose type"?